### DISCUSSION TITLE
Welcome to valgo Discussions!

### DISCUSSION THREAD

**Discussion by carlosforero** [cohesivestack/valgo#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by smgladkovskiy** [comment]
Hi!
I think it's too early to talk about a community until people get involved and start using the tool in their projects.
You'd better start with yourself. Tell us about yourself and how you got the desire to write this library. The code is already 4 years old, there is clearly a story here. I think it would be interesting for everyone to hear it. :)
